### PR TITLE
Master update

### PR DESCRIPTION
**Pull request by heckj**
updating RackHD with latest from master branches on various repos (to align with 1.1 release, proxy support, fixed microkernels, etc)

- verified fully operational with these settings and built to heckj/rackhd in ATLAS if you want to pull and view yourself

### CODE REVIEW

File: packer/template.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     {
         "type": "atlas",
         "only": ["vmware-iso"],
-        "artifact": "rackhd/rackhd",
+        "artifact": "heckj/rackhd",
```
Comment (jlongever) [inline packer/template.json]: is this the official provider?

Comment (heckj) [inline packer/template.json]: rackhd/rackhd is the office - this is work in progress, so was shimming in my personal repo so my experiments didn't break anything

**Comment by jlongever** [comment]
+1

**Comment by DavidjohnBlodgett** [comment]
:+1:

File: packer/README.md [hunk 2]
```diff
@@ -8,12 +8,17 @@ homebrew:
 
 or retrieving it from the download available at https://www.packer.io/downloads.html
 
-## To Run
+## To build locally
 
-Set up your ATLAS_USER and ATLAS_TOKEN environment variables with your username
-and API access token from Atlas, then invoke:
+    packer build -only=virtualbox-iso template.json
+    packer push -name=$ATLAS_USERNAME/rackhd template.json
```
Comment (benbp) [inline packer/README.md]: Did you mean to remove this `packer push` line here since you included it below?

Comment (heckj) [inline packer/README.md]: nope, intended to be there. You appear to always need to run one locally and manually (even with ATLAS integrations for results) before you can run in automatic mode, so left it in to be specific about ordering.

**Comment by anhou** [comment]
:+1:

**Comment by DavidjohnBlodgett** [comment]
:+1: again!

**Pull request merged**